src/parsec/include/r_gl.h

```diff
@@ -9,7 +9,13 @@
 
 // FIXME: use gles header instead of glew when compiling for OpenGL ES.
 #ifndef GLES
+//
+//  On a linux box use the system path
+#ifdef __linux__
+#include <GL/glew.h>
```

Comment:
Why is this needed? the glew header used by the project is part of the project source code (`src/libraries` should be part of the header search path).

Reply:
glew.h will be provided best by the underlying OS or for Linux by the respective distribution.
Why do you want use your own copy that is outdated.
At least on my system it is outdated.
If you can use some established well supported and maintained header/library use it. Thats all.
And I introduced that only for Linux builds as I do not know what iOS / MSWindows has in place.

Reply:
GLEW has had breaking changes over time, and we might modify it according to the project's needs. After 9 years it's not up to date anymore, but changing it affects the project's code so it's good to keep all platforms consistent when we do that. I don't think linux distros guarantee distribution of the library at all (it's not part of OpenGL itself), nor do they guarantee which version is used.

In general it's easier to manage pull requests and other changes and quicker to integrate them if they're relatively focused and self-contained.

Reply:
So than it would be easier to copy the files from fork. Instead of trying to maintain this PR.